### ISSUE TITLE
Failed to determine auto increment status

### ISSUE THREAD

**Issue by HezhiWang** [schemaspy/schemaspy#287]
Hi, 

I encountered an issue when I run schemaspy to snowflake, here is log output:

SchemaSpy generates an HTML representation of a database schema's relationships.

INFO  - Starting Main on wanghezhis-MacBook-Pro.local with PID 37583 (/Users/wanghezhi/Desktop/Knotel/schemaspy/schemaspy-6.0.0-rc2.jar started by wanghezhi in /Users/wanghezhi/Desktop/Knotel/schemaspy)
INFO  - No active profile set, falling back to default profiles: default
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.springframework.cglib.core.ReflectUtils$1 (jar:file:/Users/wanghezhi/Desktop/Knotel/schemaspy/schemaspy-6.0.0-rc2.jar!/BOOT-INF/lib/spring-core-4.3.11.RELEASE.jar!/) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of org.springframework.cglib.core.ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
INFO  - Configuration file not found
INFO  - Starting schema analysis
INFO  - Using database properties:
INFO  -   /Users/wanghezhi/Desktop/Knotel/schemaspy/snowflake.properties
INFO  - Connected to Snowflake - 2.35.1 (driver change version: 3.5.3)
INFO  - Gathering schema details
Gathering schema details...WARN  - Failed to determine auto increment status: net.snowflake.client.jdbc.SnowflakeSQLException: Cannot perform SELECT. This session does not have a current database. Call 'USE DATABASE', or use a qualified name.
WARN  - SQL: select * from DBT_HWANG."ALL_DAYS" where 0 = 1
WARN  - Unable to extract the number of rows for table ALL_DAYS, using '-1'
WARN  - net.snowflake.client.jdbc.SnowflakeSQLException: Cannot perform SELECT. This session does not have a current database. Call 'USE DATABASE', or use a qualified name.
.WARN  - Failed to determine auto increment status: net.snowflake.client.jdbc.SnowflakeSQLException: Cannot perform SELECT. This session does not have a current database. Call 'USE DATABASE', or use a qualified name.
WARN  - SQL: select * from DBT_HWANG."ZENDESK_TICKETS_XF" where 0 = 1
WARN  - Failed to determine auto increment status: net.snowflake.client.jdbc.SnowflakeSQLException: Cannot perform SELECT. This session does not have a current database. Call 'USE DATABASE', or use a qualified name.
WARN  - SQL: select * from DBT_HWANG."WAREHOUSE_OPERATION" where 0 = 1
WARN  - Unable to extract the number of rows for table ZENDESK_TICKETS_XF, using '-1'
WARN  - net.snowflake.client.jdbc.SnowflakeSQLException: Cannot perform SELECT. This session does not have a current database. Call 'USE DATABASE', or use a qualified name.
.WARN  - Failed to determine auto increment status: net.snowflake.client.jdbc.SnowflakeSQLException: Cannot perform SELECT. This session does not have a current database. Call 'USE DATABASE', or use a qualified name.

Any ideas of fixing this issue?

Thanks in advance!

**Comment by HezhiWang** [comment]
Yes, I add the specific database in the connection URL and it works fine now. 

Thanks a lot!

**Comment by sbecainfo** [comment]
Hi @HezhiWang, I'm trying to run schemaSpy against Snowflake and wondering how you got this to work. Could you share your properties file or the command line that worked in the end?
thanks,
Sebastian
